NotebookAnalyzers/utils.py

```diff
@@ -151,6 +240,18 @@ def imports_correct_position(code):
 def cells_correct_order(notebook):
     """The function takes a dict representing notebook dictionary, it returns True if the cells are executed in
     sequential order,starting from 1, and False otherwise """
+    """
+        Verifies if the notebook has been run in sequential order, starting from 1
+
+        Args:
+            notebook(dic): python dictionary object representing the jupyter notebook
+        Returns:
+            correct_exec: boolean value that is True if the Notebook has been correctly run
```

Comment:
```suggestion
            correct_exec: boolean value that is True if notebook cells have been sequentially run top to bottom.
```

Sarei più neutro qui: non è detto che l'ordine di esecuzione standard (top to bottom) sia quello **corretto** per un notebook generico (laddove per "corretto" intendo quello che permette di riprodurre senza errori l'esecuzione del notebook originale). Sicuramente sarebbe bene che lo fosse, infatti è una best practice, ma non possiamo saperlo.